core/include/cubos/core/al/audio_context.hpp

```diff
@@ -32,55 +33,58 @@ namespace cubos::core::al
     /// @ingroup core-al
     using Source = std::shared_ptr<impl::Source>;
 
-    /// @brief Possible audio formats.
-    enum class Format
-    {
-        Mono8,
-        Mono16,
-        Stereo8,
-        Stereo16,
-    };
+    /// @brief Handle to an audio listener.
+    /// @see impl::Listener - audio listener interface.
+    /// @see AudioDevice::createListener()
+    /// @ingroup core-al
+    using Listener = std::shared_ptr<impl::Listener>;
 
     /// @brief Audio device interface used to wrap low-level audio rendering APIs.
     class CUBOS_CORE_API AudioDevice
     {
     public:
-        AudioDevice() = default;
```

Comment:
This class should be put inside the impl namespace below, as is being done for the other classes (Buffer, Source, Listener)